boa_engine/src/builtins/array/mod.rs

```diff
@@ -1130,6 +1134,46 @@ impl Array {
         Ok(o.into())
     }
 
+    /// [`Array.prototype.toReversed()`][spec]
+    ///
+    /// Reverses this array, returning the result into a copy of the array.
+    ///
+    /// [spec]: https://tc39.es/ecma262/#sec-array.prototype.toreversed
+    pub(crate) fn to_reversed(
+        this: &JsValue,
+        _: &[JsValue],
+        context: &mut Context<'_>,
+    ) -> JsResult<JsValue> {
+        // 1. Let O be ? ToObject(this value).
+        let o = this.to_object(context)?;
+
+        // 2. Let len be ? LengthOfArrayLike(O).
+        let len = o.length_of_array_like(context)?;
+
+        // 3. Let A be ? ArrayCreate(len).
+        let a = Array::array_create(len, None, context)?;
+
+        // 4. Let k be 0.
+        // 5. Repeat, while k < len,
+        for i in 0..len {
+            //     a. Let from be ! ToString(𝔽(len - k - 1)).
+            let from = len - i - 1;
+
+            //     b. Let Pk be ! ToString(𝔽(k)).
+            //     c. Let fromValue be ? Get(O, from).
+            let from_value = o.get(from, context)?;
+
+            //     d. Perform ! CreateDataPropertyOrThrow(A, Pk, fromValue).
+            a.create_data_property_or_throw(i, from_value, context)
+                .expect("cannot fail per the spec");
+
+            //     e. Set k to k + 1.
```

Comment:
```suggestion
            // a. Let from be ! ToString(𝔽(len - k - 1)).
            let from = len - i - 1;

            // b. Let Pk be ! ToString(𝔽(k)).
            // c. Let fromValue be ? Get(O, from).
            let from_value = o.get(from, context)?;

            // d. Perform ! CreateDataPropertyOrThrow(A, Pk, fromValue).
            a.create_data_property_or_throw(i, from_value, context)
                .expect("cannot fail per the spec");

            // e. Set k to k + 1.
```

Remove unneeded indent